src/modules/iotjs_module_adc.c

```diff
@@ -216,16 +198,32 @@ static void iotjs_adc_close_worker(uv_work_t* work_req) {
 
 JHANDLER_FUNCTION(AdcConstructor) {
   DJHANDLER_CHECK_THIS(object);
-  DJHANDLER_CHECK_ARGS(2, object, function);
```

Comment:
Because this dumps an unreadable error when funcion is not supplied, later in the code I check the param and supply a more userfriendly message when the funcion was not passed